app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -237,38 +237,84 @@ protected function _addMagentoOrderIdToMessage($incrementId)
-        $transaction = Mage::getSingleton('core/session')->getBoltTransaction() ?: Mage::helper('boltpay/api')->fetchTransaction($reference);
-
-        $boltCartTotal = $transaction->amount->currency_symbol. ($transaction->amount->amount/100);
-        $orderTotal = $order->getGrandTotal();
-
-        $msg = Mage::helper('boltpay')->__(
-            "BOLT notification: Authorization requested for %s.  Order total is %s. Bolt transaction: %s/transaction/%s.", 
-            $boltCartTotal, $transaction->amount->currency_symbol.$orderTotal, Mage::helper('boltpay/url')->getBoltMerchantUrl(), $transaction->reference
-        );
-
-        if(Mage::getSingleton('core/session')->getWasCreatedByHook()){ // order is create via AJAX call
-            $msg .= Mage::helper('boltpay')->__("  This order was created via webhook (Bolt traceId: <%s>)", Mage::helper('boltpay/bugsnag')->getBoltTraceId());
+        catch (Exception $e) {
+            Mage::helper('boltpay/bugsnag')->notifyException($e);
         }
 
-        $order->setState(Bolt_Boltpay_Model_Payment::transactionStatusToOrderStatus($transaction->status), true, $msg)
-            ->save();
+    }
+    
+    /**
+    * Prepend an additional order info box to the gift options block.
+    * - event: core_block_abstract_to_html_after
+    *
+    * @param Varien_Event_Observer $observer
+    */
+    public function setAdminOrderViewTemplate(Varien_Event_Observer $observer) {
+        $optionsBlock = $observer->getBlock();
 
-        $order->getPayment()
-            ->setAdditionalInformation('bolt_transaction_status', $transaction->status)
-            ->setAdditionalInformation('bolt_reference', $transaction->reference)
-            ->setAdditionalInformation('bolt_merchant_transaction_id', $transaction->id)
-            ->setTransactionId($transaction->id)
-            ->save();
+        if ($optionsBlock->getNameInLayout() === 'gift_options') {
+            $customInfoBlock = Mage::app()->getLayout()->createBlock(
+                'adminhtml/template',
+                'custom_order_info',
+                array(
+                    'template' => 'boltpay/order/customernote.phtml',
+                    'order' => Mage::registry('current_order'),
+                )
+            );
+            $optionsHtml = $observer->getTransport()->getHtml();
+            $customHtml  = $customInfoBlock->toHtml();        
+            $observer->getTransport()->setHtml($customHtml . $optionsHtml);
+        }
+        elseif( get_class($optionsBlock) === Mage::getConfig()->getBlockClassName('sales/order_info') ){
+            $customInfoBlock = Mage::app()->getLayout()->createBlock('boltpay/sales_order_view_note');
+            $optionsHtml = $observer->getTransport()->getHtml();
+            $customHtml  = $customInfoBlock->toHtml(); 
+            $observer->getTransport()->setHtml(  $optionsHtml.$customHtml );
+        }
+        else{
+            return;
```

Comment:
Currently, this code breaks my admin form.  The order info box is simply not displayed.  I have not gone very deep into trouble shooting, but I would guess that we are reaching your third condition here and nothing is being returned.

I commented above that I think that this code is unnecessary and possibly undesired.  Regardless, something is happening here that prevents the standard order_info block from rendering when I try to use this code in my test environment.  I haven't looked into this extensively, but I can see that no html is rendered at the time of the order info block and execution seems to stop as if there is an un-handled error.

![image](https://user-images.githubusercontent.com/973856/44023832-d5737914-9eeb-11e8-89b1-c45283d260a4.png)